lib/connect.c

```diff
@@ -866,7 +866,7 @@ CURLcode Curl_is_connected(struct Curl_easy *data,
   int error = 0;
   struct curltime now;
   int rc = 0;
-  unsigned int i;
+  int i;
```

Comment:
why?

Reply:
const int other = i ^ 1;

Reply:
and why does that get better if `i` is not unsigned?

Reply:
Because I only goes up to 1 anyway. It silences warnings, assuming you intended to use the variable like a Boolean.

Reply:
What warning? There's nothing wrong there, having a warning for that seems overzealous.